packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -2579,7 +2674,11 @@ function commitReconciliationEffects(finishedWork: Fiber) {
   const flags = finishedWork.flags;
   if (flags & Placement) {
     try {
-      commitPlacement(finishedWork);
+      if (enableFloat && finishedWork.tag === HostSingleton) {
+        commitSingletonPlacement(finishedWork);
```

Comment:
The `Placement` flag is supposed to be used on the inserted Fiber, which might be a parent Component, not on the HostComponent itself.

This should be done with an `Update` flag instead.